### DISCUSSION TITLE
What is this message?

### DISCUSSION THREAD

**Discussion by jcodyparker** [ORelio/Minecraft-Console-Client#2361] [Q&A]
What is this message and how do I stop it from happening? Every 30 seconds it posts this into the console

![image](https://user-images.githubusercontent.com/90956620/206348271-e1dca2ea-f569-41db-97fb-c0d411c5ba86.png)

**Comment by milutinke** [comment]
> What is this message and how do I stop it from happening? Every 30 seconds it posts this into the console
> 
> ![image](https://user-images.githubusercontent.com/90956620/206348271-e1dca2ea-f569-41db-97fb-c0d411c5ba86.png)

We currently don't have a parser for new JSON encoded messages that support new colors and gradients, you can't hide it.
